### PR TITLE
Contact Form and Finish Mobile Version

### PR DESCRIPTION
**Pull request by manzitresor**
This Part contains Contact Form of Portfolio Project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -187,5 +187,21 @@ <h3 class="conn">LET’S CONNECT</h3>
...
+        <form class="contact-form" action="https://formspree.io/f/xeqwngzr" method="POST">
+            <input type="text" name="name" placeholder="Lorem Ipsum" placeholder="Name" maxlength="30" required>
+            <input type="email" name="email" placeholder="loremipsum@gmail.com" placeholder="Email" required>
+            <textarea  name="message" maxlength="500" required>
+                Write your message here...
+            </textarea>
+
+            <button class="contact-btn" type="submit">Get in touch</button>
+        </form>
+    </section>
```
Comment (bobb-Rob) [inline index.html]: - [ ] [REQUIRED] The margin top between the contact section and the preceeding section is much more than you have in your design at the moment. Please ensure you stick to the design template details as much as possible. 

Figma design | Your work
---- | ----
image | image

File: index.html [hunk 2]
```diff
@@ -187,5 +187,21 @@ <h3 class="conn">LET’S CONNECT</h3>
...
+        <h2 class="contact-heading">Contact me</h2>
+        <p class="contact-content">
+            If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it!
+        </p>
+        <form class="contact-form" action="https://formspree.io/f/xeqwngzr" method="POST">
+            <input type="text" name="name" placeholder="Lorem Ipsum" placeholder="Name" maxlength="30" required>
+            <input type="email" name="email" placeholder="loremipsum@gmail.com" placeholder="Email" required>
+            <textarea  name="message" maxlength="500" required>
+                Write your message here...
+            </textarea>
```
Comment (bobb-Rob) [inline index.html]: - [ ] [REQUIRED] This text on line 200 should be a placeholder and not a value of the `textarea` element. Please use the `placeholder` attribute to set the text.

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️
Hi @manzitresor, 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you are almost there!  😃 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by victorokoroji — APPROVED** [review]
Hi @manzitresor, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- Required changes implemeted✅ 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**